externals/kyuubi-spark-sql-engine/src/test/scala/org/apache/kyuubi/engine/spark/operation/SparkOperationSuite.scala

```diff
@@ -746,4 +751,96 @@ class SparkOperationSuite extends WithSparkSQLEngine with JDBCTests {
       assert(tFetchResultsResp.getStatus.getStatusCode === TStatusCode.SUCCESS_STATUS)
     }
   }
+
+  test("SPARK-26533: Support query auto timeout cancel on thriftserver - setQueryTimeout") {
```

Comment:
we need to move these test to the superclass to test both the server and engine.